lib/ios/RNNNavigationOptions.m

```diff
@@ -50,6 +54,24 @@ -(void)applyOn:(UIViewController*)viewController {
 		if (self.topBar.title) {
 			viewController.navigationItem.title = self.topBar.title;
 		}
+
+		if (self.topBar.largeTitle){
+			if ([self.topBar.largeTitle boolValue]) {
+				if (@available(iOS 11.0, *)) {
+					viewController.navigationController.navigationBar.prefersLargeTitles = YES;
```

Comment:
I'm sure this can be cleaned up somehow.. do all iOS11 things inside a iOS11 function..

Reply:
instead of asking several times.. ask once and do in another function